docs/index.md

```diff
@@ -1,11 +1,58 @@
+
+- [The earthaccess Python Library: Awesome Discovery and Access of NASA Earthdata](https://docs.google.com/presentation/d/e/2PACX-1vT8BN1C4FkHrid_eVfOwJjZzKGn9Nomb6mnDj3Nld-nD0ekzoenxQIq6Xzo9VFQsXbixCNskadBwPwM/pub?start=false&loop=false&delayms=3000) - Amy Steiker, Andrew P. Barrett, Charles Daniels, Matthew Fisher, Daniel Kaufman, Joseph H. Kennedy, Luis López, Julia S. Stewart Lowndes, Jessica Scheick, The NASA Openscapes mentors team. December 9, 2024. AGU Fall Conference Poster ([slide](https://docs.google.com/presentation/d/1iBqeMQcdNAxQfhIH9jdcvI2m1HmBxJFixLX1BnwG-S8/edit#slide=id.g317ff1cef8c_1_87))
+
+
+# About
+
+![*NASA Earthdata using `earthaccess` for impactful science. Art by Allison Horst and Openscapes.*](images/earthaccess_environmental_horst.jpg){alt="3-panel watercolor artwork depicting earthaccess' position as a missing link between environmental data and data analysis. Depicts environmental data within mountains and streams with arrows labeled 'import' and 'earthaccess' that connect to a data analysis panel and then a communicate panel." width="100%"}
+
+We can use `earthaccess` to do impactful science, faster:
+
+- Search Data: Explore NASA's extensive collection of Earth science data, powered by [CMR](https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html).
+- Access Data: Download data to your local machine or stream it directly into your Python scripts through [`fsspec`](https://github.com/fsspec/filesystem_spec).
+- Authenticate: Manage your [NASA Earthdata Login (EDL)](https://urs.earthdata.nasa.gov) and Earthdata Cloud credentials.
 
-During several workshops organized by [NASA Openscapes](https://nasa-openscapes.github.io/events.html), the need to provide easy-to-use tools to our users became evident. Open science is a collaborative effort; it involves people from different technical backgrounds, and the data analysis to solve the pressing problems we face cannot be limited by the complexity of the underlying systems. Therefore, providing easy access to NASA Earthdata regardless of the data storage location (hosted within or outside of the cloud) is the main motivation behind this Python library.
 
 The library is an open source community effort under an [MIT license](LICENSE.txt).  We welcome contributions to improve `earthaccess`.  Please see the [Contributing Guide](contributing/index.md) to learn how to get involved.
 
-***earthaccess*** handles authentication with [NASA's Earthdata Login (EDL)](https://urs.earthdata.nasa.gov), search using NASA's [CMR](https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html) and access through [`fsspec`](https://github.com/fsspec/filesystem_spec).
+
+
+# Supported By
+
+<p align="center">
+<img alt="NASA logo" src="nasa-logo.png" width="40%" />
+<img alt="ESDIS logo" src="logos/earthdata-logo.jpg" width="40%" />
```

Comment:
Personal opinion: I don't think NASA's logos should be larger than the other logos.

Reply:
The ESDIS logo, IMO, is kind of meaningless. Few people will recognize it, I think! I didn't. My first thought was "is that a weird version of the pangeo logo I haven't seen before"?

Reply:
I think it may be worth considering replacing the logos with a bulleted list. I don't think we're going to find a reasonable way to display these logos with very different shapes in a way that doesn't represent some (OB.DAAC, Coiled) as significantly smaller than others.

Reply:
> I think it may be worth considering replacing the logos with a bulleted list.

Your point is well taken, but I think the images over text piece is critical here. I've made some edits to the image order to take a stab at improving the display without getting too fancy. I think there is a lot of space for improvement here (e.g. adding links to the images), but it's not the most pressing place to invest time at the moment.